src/components/DialogHeader/DialogHeader.tsx

```diff
@@ -0,0 +1,60 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React, { ReactElement } from 'react';
+import { merge } from '@utilities/merge';
+import { IconCross } from '@foundation/Icon';
+import { Badge, BadgeProps } from '@components/Badge';
+
+export type DialogHeaderProps = {
+    title: string;
+    size?: DialogHeaderSize;
+    collapseBottom?: boolean;
+    close?: boolean;
+    onClose?: () => void;
+    icon?: ReactElement;
+    badge?: BadgeProps;
+};
+
+export enum DialogHeaderSize {
+    Null = 'Null',
+    Large = 'Large',
+}
+
+export const DialogHeader = ({
+    title,
+    collapseBottom = false,
```

Comment:
collapseBottom? What is the point of this?

Reply:
collapseBottom removes the border and spacing below the header so you could use tabs for example. It's in the Figma file.

Reply:
IMHO I would remove that property and other stuff,  as DialogHeader should only be responsible for styling e.g. padding or w/e, and leave it to the end user to compose what Header should have via children. 

In this case: 

```
<DialogHeader>
   <Collapsable in={expanded} ...>
     ...
   </Collapsable>
</DialogHeader>

```

Reply:
We provide A DialogHeader component with strict styling. You can't pass children. That's what Designers would like to see in all overlays components (including Modal) But it's optional to use it. You can pass a Custom Header to the Dialog component if you want to. Same with Body and Footer. If it happens Designers use the Fondue Designs, it's ready for you to use.